ert3/algorithms/_sensitivity.py

```diff
@@ -55,29 +77,48 @@ def one_at_a_time(
     if len(parameters) == 0:
         raise ValueError("Cannot study the sensitivity of no variables")
 
-    # The lower extremal value of the analysis. Each variable will after turn
+    # The lower extremal value of the analysis. Each variable will in turn
     # be explored as ppf(q) and ppf(1-q) as the two extremal values.
+
     if tail is not None:
         q = (1 - tail) / 2
     else:
         q = (1 - 0.99) / 2
 
     evaluations = []
-    for group_name, dist in parameters.items():
+    for parameter_name, dist in parameters.items():
         lower = dist.ppf(q)
         upper = dist.ppf(1 - q)
-        const_records = set(parameters.keys())
-        const_records.remove(group_name)
+        constant_records = set(parameters.keys())
+        constant_records.remove(parameter_name)
+
+        parameter_index: Union[Tuple[str, ...], Tuple[int, ...]]
+        if parameters[parameter_name].is_scalar:
+            parameter_index = ("dummy",)
```

Comment:
Could you explain this with an inline comment?

Reply:
Code is superfluous, will cleanup!